lib/core/test/unit/Cardano/Wallet/DB/SqliteFileModeSpec.hs

```diff
@@ -131,7 +133,9 @@ spec =  do
             unsafeRunExceptT $ createWallet db testWid testCp testMetadata
             unsafeRunExceptT $ putTxHistory db testWid (Map.fromList testTxs)
             destroyDBLayer ctx
-            testOpeningCleaning f (`readTxHistory` testWid) testTxs mempty
+            testOpeningCleaning f
+                (\db' -> readTxHistory db' testWid defaultTxSortOrder wholeRange)
+                testTxs mempty
```

Comment:
Minor remark but, when formatting code multiline, try to avoid putting several arguments on the same line. It reads better to have on argument per line:

```hs
            testOpeningCleaning 
                f
                (\db' -> readTxHistory db' testWid defaultTxSortOrder wholeRange)
                testTxs 
                mempty
```

rule of thumb is really: either they all fit on a line, or there's one argument per line.

--> Coding Standard maybe?